src/style/styles.scss

```diff
@@ -1,4 +1,32 @@
 @import '_reset';
-@import 'FiveDay';
+@import '_variables';
+@import '_homepage';
+@import '_header-style';
 @import '_settings';
-@import 'header-style';
+@import '_five-day';
+
+html {
+  box-sizing: border-box;
+}
+
+*, *:before, *:after {
+  box-sizing: inherit;
+}
+
+body {
+  background-color: $color-main-background;
+  font-family: 'Open Sans';
+}
+
+body,
+.root,
+.App {
```

Comment:
will fix, was going to go through all the sass again if we got all the react and redux working to consolidate and clean up.